mappings/net/minecraft/entity/projectile/ShulkerBulletEntity.mapping

```diff
@@ -12,7 +13,7 @@ CLASS net/minecraft/unmapped/C_moyveclb net/minecraft/entity/projectile/ShulkerB
 		ARG 3 target
 		ARG 4 axis
 	METHOD m_nkokkcwc getDirection ()Lnet/minecraft/unmapped/C_xpuuihxf;
-	METHOD m_orsuvazc (Lnet/minecraft/unmapped/C_xpuuihxf$C_rmpfouoz;)V
+	METHOD m_orsuvazc selectNextMoveDirection (Lnet/minecraft/unmapped/C_xpuuihxf$C_rmpfouoz;)V
```

Comment:
selectMoveDirection

Reply:
Disagree, this is a calculation for the next move direction the shilker bullet takes, exicitly next

Reply:
Ah okay